install-nhc.sh

```diff
@@ -70,12 +70,12 @@ distro_check=$( cat /etc/os-release | grep -i ID_LIKE=)
 distro=`awk -F= '/^NAME/{print $2}' /etc/os-release`
 
 if [[ $distro_check =~ "debian" ]]; then
-  sudo DEBIAN_FRONTEND=noninteractive apt-get install -y libpci-dev hwloc build-essential libboost-program-options-dev libssl-dev plocate cmake
+  sudo DEBIAN_FRONTEND=noninteractive apt-get install -y libpci-dev hwloc build-essential libboost-program-options-dev libssl-dev cmake $(apt-cache search libpci-dev | grep -o '^[^ ]*')
 elif [[ $distro =~ "AlmaLinux" ]]; then
-  sudo dnf install -y pciutils-devel hwloc openssl-devel boost-devel mlocate cmake
+  sudo DEBIAN_FRONTEND=noninteractive apt-get install -y pciutils-dev hwloc libssl-dev libboost-dev cmake
```

Comment:
Apt-get does not work on AlmaLinux. Please revert to the old way of using dnf.

add whatever additional packages to the existing install line (previously line 75)

Please test this change on AlmaLinux

Reply:
Obviously remove any references to mlocate/plocate

Reply:
Removed locate as discussed